packages/packages/recently-edited/src/hooks/use-add-new-page.ts

```diff
@@ -0,0 +1,24 @@
+import apiFetch from '@wordpress/api-fetch';
+import { useCallback } from 'react';
+export interface NewPost {
+	id: number,
+	edit_url: string,
+}
+export default function useAddNewPage() {
+	const createNewPage = useCallback( () => {
+		addNewPage().then( ( newPost ) => {
+			if ( 0 !== newPost.id ) {
+				window.location.href = newPost.edit_url;
+			}
+		} );
+	}, [] );
+
+	return createNewPage;
+}
```

Comment:
useCallback is unnecessary because its not using any state, therefor the entire custom hook is unnecessary